src/librbd/cache/pwl/SSDLogEntry.cc

```diff
@@ -0,0 +1,54 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#include "librbd/cache/ImageWriteback.h"
+#include "librbd/cache/pwl/SSDLogEntry.h"
+
+#define dout_subsys ceph_subsys_rbd_pwl
+#undef dout_prefix
+#define dout_prefix *_dout << "librbd::cache::pwl::SSDWriteLogEntry: " << this << " " \
+                           <<  __func__ << ": "
+
+namespace librbd {
+
+namespace cache {
+
+namespace pwl {
+
+void SSDWriteLogEntry::init_cache_bl(bufferlist &src_bl, uint64_t off, uint64_t len) {
+  pmem_bl.clear();
```

Comment:
Nit: `pmem` is not a good variable name in this context

Reply:
`pmem_bl` variable is shared by both SSD and RWL. Could rename it to `cache_bl` -- does that sound ok?

Reply:
Sure -- that would be fine by me. I'm just trying to avoid incorrect naming.

Reply:
Done